### DISCUSSION TITLE
individual cell borders

### DISCUSSION THREAD

**Discussion by BrianHung** [glideapps/glide-data-grid#477] [Ideas]
Allow cells to define their own custom borders as part of [`BaseGridCell` or `BaseGridCell.themeOverride`](https://github.com/glideapps/glide-data-grid/blob/main/packages/core/API.md#gridcell). 

For reference, Google Sheets supports this formatting by allowing a border property on cell which has nested top, left, bottom, right values: https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/cells

Technically, I think developers could do this by [overriding cell rendering behavior](https://github.com/glideapps/glide-data-grid/blob/main/packages/core/API.md#drawcell) but it would be nice if it was supported out of the box / in core.

**Comment by jassmith** [comment]
This is quite challenging from a technical standpoint. There are a couple major issues at play.

1) If cell A1 has a green border and cell A2 has a red border what color should the border between them be?
2) This will definitely slow down drawing significantly as cell borders must be drawn by the cell renderer and not in a single pass.
